### PR TITLE
Updated various command functions

### PR DESCRIPTION
**Pull request by clawdius**
- Added "buyar-sek" command to delete the current running session
- Prevent user to create more than 1 session
- Prevent user to delete non-existing session
- Fixed the _ApplicationCommandCreate_ function using the User ID, now it uses the Application ID

### CODE REVIEW

File: handlers/gaming-session/create-gaming-session.go [hunk 1]
```diff
@@ -107,6 +124,38 @@ func DeclineGamingSession(s *discordgo.Session, i *discordgo.InteractionCreate)
 	}
 }
 
+func DeleteGamingSession(s *discordgo.Session, i *discordgo.InteractionCreate) {
```
Comment (bismastr) [inline handlers/gaming-session/create-gaming-session.go:127]: deleteFunction should be on seperate handlers file.

**Review by bismastr — DISMISSED** [review]

**Review by bismastr — CHANGES_REQUESTED** [review]
Need changes

**Comment by clawdius** [comment]
Handlers now split into 4 distinct functions, including separating **deleteFunction** to it's own handler, along with the **updateFunction** and **utilityFunction** for easy maintenance.

**Review by bismastr — APPROVED** [review]
LGTM

**Pull request merged**